package.json

```diff
@@ -81,7 +81,8 @@
                         "bat": "",
                         "shellscript": "bash",
                         "fsharp": "fsi",
-                        "csharp": "scriptcs"
+                        "csharp": "scriptcs",
+                        "vbscript": "cscript //Nologo"
```

Comment:
Exactly, I had to install the VBScript language extension to get the language entry.

It is rather inconvenient to manually switch the file, so a config override would be a nice addition.  Although, I'm curious why .vbs is mapped to Visual Basic in the VSCode 'vb' definition.  Additionally, should the VBScript language extension override the .vbs extension mapping.  I'm not too familiar with VSCode language server implementation so I could be making incorrect assumptions.